node/status_node.go

```diff
@@ -178,14 +180,40 @@ func (n *StatusNode) setupRPCClient() (err error) {
 }
 
 func (n *StatusNode) discoveryEnabled() bool {
-	return n.config != nil && !n.config.NoDiscovery && n.config.ClusterConfig != nil
+	return n.config != nil && !n.config.NoDiscovery && n.config.ClusterConfig != nil && len(n.config.EnabledDiscoveries) != 0
 }
 
 func (n *StatusNode) startDiscovery() error {
-	n.discovery = peers.NewDiscV5(
-		n.gethNode.Server().PrivateKey,
-		n.config.ListenAddr,
-		parseNodesV5(n.config.ClusterConfig.BootNodes))
+	if len(n.config.EnabledDiscoveries) != 1 {
+		return errors.New("only one discovery can be used (will be allowed to use more in next change)")
+	}
+	switch dtype := n.config.EnabledDiscoveries[0]; dtype {
+	case discovery.EthereumV5:
+		n.discovery = discovery.NewDiscV5(
+			n.gethNode.Server().PrivateKey,
+			n.config.ListenAddr,
+			parseNodesV5(n.config.ClusterConfig.BootNodes))
+	case discovery.RendezvousV1:
+		if len(n.config.ClusterConfig.RendezvousNodes) == 0 {
```

Comment:
`RendezvousNodes` are just regular nodes with rendezvous support, right?

Reply:
no. rendezvous nodes are servers from https://github.com/status-im/rendezvous/blob/master/cmd/server/main.go . so rendezvous node is an analogue to bootnode. we can  add rendezvous server support to every statusd node in future if we want that

Reply:
Ah, I see. Thanks!